docs-v2/pages/connect/index.mdx

```diff
@@ -73,14 +73,6 @@ All credentials and tokens are sent to Pipedream securely over HTTPS, and encryp
 - **Use secure, session-based auth between your client and server** — authorize all requests from your client to your server using a secure, session-based auth mechanism. Use well-known identity providers with services like [Clerk](https://clerk.com/), [Firebase](https://firebase.google.com/), or [Auth0](https://auth0.com/) to securely generate and validate authentication tokens. The same follows for Pipedream workflows — if you trigger Pipedream workflows from your client or server, validate all requests in the workflow before executing workflow code.
 - **Secure your workflows** — See our [standard security practices](/privacy-and-security/best-practices) for recommendations on securing your Pipedream workflows.
 
-## Glossary of terms
```

Comment:
Can we keep this and just put it down bottom? I believe we still link to this section elsewhere